librz/core/cmd_analysis.c

```diff
@@ -8088,6 +8088,111 @@ RZ_IPI RzCmdStatus rz_analysis_rtti_demangle_class_name_handler(RzCore *core, in
 	return RZ_CMD_STATUS_ERROR;
 }
 
+RZ_IPI RzCmdStatus rz_analysis_print_global_variable_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	char *var_name = NULL;
+	if (argc == 2) {
+		var_name = (char *)argv[1];
+	}
+	rz_analysis_var_global_list_show(core->analysis, state, var_name);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analysis_global_variable_add_handler(RzCore *core, int argc, const char **argv) {
+	if (argc < 4) {
+		return RZ_CMD_STATUS_ERROR;
+	}
+	char *var_name = (char *)argv[1];
+	ut64 addr = rz_num_math(core->num, (char *)argv[2]);
+	char *type = (char *)argv[3];
+	char *comment = NULL;
+	if (argc == 5) {
```

Comment:
What happens if there are 6 arguments? It's probably add a check for `argc > 5` at the beginning too.

Reply:
Yes, I agree